{"uid":"225a43481c453d5d","name":"Test Case 16 : Client Review Visibility In Reviews Page","fullName":"Store_Reviews.Reviews.test_review.TestReview#test_client_reviews_visibility","historyId":"8b1667338f1db0d480f8d00ed77452d4","time":{"start":1689163486421,"stop":1689163486653,"duration":232},"descriptionHtml":"\n    <br><b>Test Case Description:</b>\n    <br><br>Validate that Reviews list page in the banner \"Clinte Review\" text is present or not?\n    \n    <br><br><b>Expeceted Results:</b>\n    <br><br>\"Clinte Review \" text should be present in the banner of reviews list page.\n    \n    ","status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"init_driver","time":{"start":1689163434025,"stop":1689163435192,"duration":1167},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"testStage":{"descriptionHtml":"\n    <br><b>Test Case Description:</b>\n    <br><br>Validate that Reviews list page in the banner \"Clinte Review\" text is present or not?\n    \n    <br><br><b>Expeceted Results:</b>\n    <br><br>\"Clinte Review \" text should be present in the banner of reviews list page.\n    \n    ","status":"passed","steps":[{"name":"Checking Client Reviews visibility","time":{"start":1689163486421,"stop":1689163486653,"duration":232},"status":"passed","steps":[{"name":"Taking Body Of Reviews Page","time":{"start":1689163486421,"stop":1689163486653,"duration":232},"status":"passed","steps":[{"name":"Taking Text from element","time":{"start":1689163486421,"stop":1689163486653,"duration":232},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"by_locator","value":"('xpath', '//body')"}],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":true},{"name":"Client Reviews Is Visible","time":{"start":1689163486653,"stop":1689163486653,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"attachments":[],"parameters":[],"stepsCount":2,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":true}],"attachments":[],"parameters":[],"stepsCount":3,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":true}],"attachments":[],"parameters":[],"stepsCount":4,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":true},"afterStages":[{"name":"init_driver::0","time":{"start":1689163550869,"stop":1689163550871,"duration":2},"status":"broken","statusMessage":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=36313): Max retries exceeded with url: /session/da70a28d2ea10d4c4f9feb53c90ed148/window (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f312aefc250>: Failed to establish a new connection: [Errno 111] Connection refused'))\n","statusTrace":"  File \"/home/kishorekumarmahto/.local/lib/python3.10/site-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/home/kishorekumarmahto/.local/lib/python3.10/site-packages/_pytest/fixtures.py\", line 911, in _teardown_yield_fixture\n    next(it)\n  File \"/home/kishorekumarmahto/my_backup/Testing/Dr._Garcia_Biomagnetism_website/Tests/conftest.py\", line 20, in init_driver\n    web_driver.close()\n  File \"/usr/local/lib/python3.10/dist-packages/selenium/webdriver/remote/webdriver.py\", line 561, in close\n    self.execute(Command.CLOSE)\n  File \"/usr/local/lib/python3.10/dist-packages/selenium/webdriver/remote/webdriver.py\", line 442, in execute\n    response = self.command_executor.execute(driver_command, params)\n  File \"/usr/local/lib/python3.10/dist-packages/selenium/webdriver/remote/remote_connection.py\", line 294, in execute\n    return self._request(command_info[0], url, body=data)\n  File \"/usr/local/lib/python3.10/dist-packages/selenium/webdriver/remote/remote_connection.py\", line 316, in _request\n    response = self._conn.request(method, url, body=body, headers=headers)\n  File \"/usr/lib/python3/dist-packages/urllib3/request.py\", line 74, in request\n    return self.request_encode_url(\n  File \"/usr/lib/python3/dist-packages/urllib3/request.py\", line 96, in request_encode_url\n    return self.urlopen(method, url, **extra_kw)\n  File \"/usr/lib/python3/dist-packages/urllib3/poolmanager.py\", line 375, in urlopen\n    response = conn.urlopen(method, u.request_uri, **kw)\n  File \"/usr/lib/python3/dist-packages/urllib3/connectionpool.py\", line 783, in urlopen\n    return self.urlopen(\n  File \"/usr/lib/python3/dist-packages/urllib3/connectionpool.py\", line 783, in urlopen\n    return self.urlopen(\n  File \"/usr/lib/python3/dist-packages/urllib3/connectionpool.py\", line 783, in urlopen\n    return self.urlopen(\n  File \"/usr/lib/python3/dist-packages/urllib3/connectionpool.py\", line 755, in urlopen\n    retries = retries.increment(\n  File \"/usr/lib/python3/dist-packages/urllib3/util/retry.py\", line 574, in increment\n    raise MaxRetryError(_pool, url, error or ResponseError(cause))\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true}],"labels":[{"name":"tag","value":"@pytest.mark.usefixtures('init_driver')"},{"name":"parentSuite","value":"Store_Reviews.Reviews"},{"name":"suite","value":"test_review"},{"name":"subSuite","value":"TestReview"},{"name":"host","value":"isadmin"},{"name":"thread","value":"85329-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"Store_Reviews.Reviews.test_review"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":true,"retry":true,"extra":{"categories":[],"tags":["@pytest.mark.usefixtures('init_driver')"]},"source":"225a43481c453d5d.json","parameterValues":[]}